### PR TITLE
Fix for #7125, makes error messages clearer.

### LINKED ISSUE #7125: Filter Arrays with filter conflicting statements in instructions and test
Challenge [Filter Arrays with filter](http://www.freecodecamp.com/challenges/filter-arrays-with-filter#?solution=var%20oldArray%20%3D%20%5B1%2C2%2C3%2C4%2C5%2C6%2C7%2C8%2C9%2C10%5D%3B%0A%0A%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%0Avar%20newArray%20%3D%20oldArray.filter(function(val)%7B%0A%20%20return%20val%20%3C%206%3B%0A%7D)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.109 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

screen shot 2016-02-18 at 8 30 22 pm 2

Conflicting statements instructions say 

Use filter to create a new array with all the values from oldArray which are less than 6.
oldArray should not change.

Test say 
You should have filtered out all values from the array that are greater than 5.

My code:
```javascript
var oldArray = [1,2,3,4,5,6,7,8,9,10];

// Only change code below this line.

var newArray = oldArray.filter(function(val){
  return val < 6;
});

```

### PR DESCRIPTION
**Pull request by Zexbe**
Changes the error messages in Filter Arrays with filter challenge. Makes it clear that filter is creating a new array.

closes #7125

### CODE REVIEW

**Comment by garbark** [comment]
I haven't been login in about two hours so it's somebody else thank you
have a beautiful night

Gary M Barker

**Comment by bugron** [comment]
Closing this PR because it is not following our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md). Please read it, and then follow steps from our pull request template and only after that open a PR. Thanks.

**Comment by jamigibbs** [comment]
@bugron Can you be more specific to the OP about what they need to change?

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -404,9 +404,9 @@
       ],
       "tests": [
```
Comment (panyarerkcap) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: [[5×5/5]]

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 2]
```diff
@@ -404,9 +404,9 @@
...
         "assert.deepEqual(oldArray, [1,2,3,4,5,6,7,8,9,10], 'message: You should not change the original array.');",
-        "assert.deepEqual(newArray, [1,2,3,4,5], 'message: You should have filtered out all values from the array that are greater than 5.');",
-        "assert(editor.getValue().match(/array\\.filter\\s*\\(/gi), 'message: You should be using the <code>filter</code> method to remove the values from the array.');",
-        "assert(editor.getValue().match(/\\[1\\,2\\,3\\,4\\,5\\,6\\,7\\,8\\,9\\,10\\]/gi), 'message: You should only be using <code>filter</code> to modify the contents of the array.');"
+        "assert.deepEqual(newArray, [1,2,3,4,5], 'message: You should have filtered out all the values from the array that are greater than 5.');",
+        "assert(editor.getValue().match(/array\\.filter\\s*\\(/gi), 'message: You should be using the <code>filter</code> method to create a new array.');",
+        "assert(editor.getValue().match(/\\[1\\,2\\,3\\,4\\,5\\,6\\,7\\,8\\,9\\,10\\]/gi), 'message: You should only be using <code>filter</code> to make the new array.');"
```
Comment (panyarerkcap) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: 9a7faad1

**Pull request closed without merging**